index.bs

```diff
@@ -4793,10 +4822,10 @@ worker=] algorithm:
 
 1. Let |realm id| be the [=realm id=] for |realm|.
 
-1. Let |params| be a map mathcing the <code>RealmDestroyedParameters</code>
+1. Let |params| be a map mathcing the <code>script.RealmDestroyedParameters</code>
```

Comment:
```suggestion
1. Let |params| be a map matching the <code>script.RealmDestroyedParameters</code>
```

Looks like this went in due to c&p. There is one more instance that you might want to fix as well?